src/test/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+
+public class Main {
+    private static final int FIGURES_LENGTH = 3;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[FIGURES_LENGTH * 2];
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = new FigureSupplier().getRandomFigure();
+            figures[i + figures.length / 2] = new FigureSupplier().getDefaultFigure();
+        }
+        for (int i = 0; i < figures.length; i++) {
+            figures[i].getDraw();
+        }
```

Comment:
```suggestion
        Figure[] figures = new Figure[FIGURES_LENGTH * 2];
        FigureSupplier figureSupplier = new FigureSupplier();
        for (int i = 0; i < figures.length / 2; i++) {
            figures[i] = figureSupplier.getRandomFigure();
            figures[i + figures.length / 2] = figureSupplier.getDefaultFigure();
        }
        for (int i = 0; i < figures.length; i++) {
            figures[i].getDraw();
        }
```

lets not create lots of rebundant FigureSupplier instances

Reply:
what about this comment?